src/Pages/Product/Product.js

```diff
@@ -9,62 +11,119 @@ export class Product extends Component {
     this.state = {
       productDetailImages: [],
       productInformation: {},
+      productReviewData: [],
+      quantity: 1,
+      moreView: false,
     };
   }
 
   componentDidMount() {
-    fetch('/data/wearImagesData.json')
+    fetch('/data/electronicImagesData.json')
       .then(res => res.json())
       .then(productPageData => {
         this.setState({
           productDetailImages: productPageData,
         });
       });
+
+    // fetch('http://192.168.0.4:8000/products/440')
+    //   .then(res => res.json())
+    //   .then(productInfoData => {
+    //     this.setState({
+    //       productInformation: productInfoData.data.product,
+    //     });
+    //   });
+
     fetch('/data/productInfoData.json')
       .then(res => res.json())
       .then(productInfoData => {
         this.setState({
-          productInformation: productInfoData,
+          productInformation: productInfoData.data.product,
+        });
+      });
+
+    fetch('/data/reviewData.json')
+      .then(res => res.json())
+      .then(reviewData => {
+        this.setState({
+          productReviewData: reviewData,
         });
       });
   }
 
+  plusQuantity = e => {
+    console.log(e);
+    this.setState({
+      quantity: Number(this.state.quantity) + 1,
+    });
+  };
+
+  minusQuantity = () => {
+    if (this.state.quantity > 1) {
+      this.setState({
+        quantity: this.state.quantity - 1,
+      });
+    }
+  };
```

Comment:
```suggestion
  minusQuantity = () => {
    if (this.state.quantity === 1) return;
    
    this.setState({ quantity: this.state.quantity - 1 });
  };
```
early return 패턴을 사용하면 if 문으로 인한 depth 증가를 피할 수 있습니다!

Reply:
오오 훨씬 간결하군용@